### PR TITLE
Demon Prison District 1~5 - Critical bug!

### PR DESCRIPTION
**Pull request by Sunaries**
@imcgames I have encountered 2 important bugs (one is shown on video) and I have a request.

**REQUEST**

Currently when viewing another character's profile, it displays the following as example: **Circle 1 Fencer Level 171**
In English this is a bit odd. It can be simplified to: 

GitHub Logo

↑ Code: Level {#ffff00}%s{/} %s (Circle {#ffff00}%s{/}) - (_UI_20150317_000157_)
But there is a problem. As you can see, the client recognizes the first parameter '%s' as 랭크/서클 (1) and third parameter '%s' as 레벨 (171). 

The design you see in the image is much simplified and looks better. It would be great to have parameters switched or to have {level} and {circle}/{rank} parameters, so it is easier for the English ver.

Thank you.

**BUG**

**1.** The HP of Dionysus (디오니스) in the Demon Prison map is glitched or has low HP. I defeated the boss monster in less than 5 seconds. I do not think that this monster is supposed to be defeated that quickly.

**2.**

I cannot continue right now because of a gamebreaking bug. I am to the point to seal the Demon Lord Hauberk. I accept the quest and the cutscene appears. The 3 Kupoles are waiting for me to seal Hauberk. When I defeat Hauberk (very easy boss..) he disappears and says one final thing. Then the cutscene UI stays and I have no idea what to do next.

I tried to defeat Hauberk slowly, that didn't help. I abandoned the quest and re-started it, that didn't help either.

I'm currently stuck, because I cannot proceed with the game.

Video: https://www.youtube.com/watch?v=sZfGi2MXokU

**3.**

The following lines from Goddess Vakarine still appear in Korean despite that they are translated in the Demon Prison story line.

QUEST_20151001_002568   구원자여.{nl}보시다시피 지금의 저는 혼자의 힘으론 아무것도 할 수 없는 처지입니다.{nl}그대들이 말하는 신수의 날 이후로 미약한 힘 만이 남아있을 뿐..{nl}
QUEST_20151001_002569   이 수감소에서는는 새로운 재앙이 움트고 있습니다.{nl}마족들은 계속해서 전이의 틈새로 넘어와 이 수감소를 새로운 거점으로 만드려 하고 있지요.{nl}
QUEST_20151001_002570   저에겐 전이의 틈새를 닫을 힘 조차 남아있지 않습니다만.{nl}그렇다고 해서 아주 방법이 없는 건 아닙니다.{nl}
QUEST_20151001_002571   바로 그대의 도움이 이 상황을 뒤집을 수 있는 유일한 돌파구이지요.{nl}부디 무력한 절 대신하여 전이의 틈새를 닫을 힘을 빌려주십시오.

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

**Comment by vtange** [comment]
That's really a bug.. It's supposed go "Wriiing!" with the Quest Completed thing once you defeat Hauberk

**Comment by Sunaries** [comment]
@imcgames I just want to add that I have resolved the bug of Hauberk. 

**My solution**: I took out my companion from the character login screen and joined the world. Then I started to accept the quest again and this time I slowly attacked instead of attacking it fast. After his HP was around 25%, a magic circle appeared and a speech bubble and he was gone and the cutscene finally ended. 

Please note this to your development team. I have no doubt that users want to deal high damage at a very fast rate at boss monsters, so this should be no exception.

**Comment by Sunaries** [comment]
Good to merge.

@imcgames I just want to add that I have resolved the bug of Hauberk. 

**My solution**: I took out my companion from the character login screen and joined the world. Then I started to accept the quest again and this time I slowly attacked instead of attacking it fast. After his HP was around 25%, a magic circle appeared and a speech bubble and he was gone and the cutscene finally ended. 

Please note this to your development team. I have no doubt that users want to deal high damage at a very fast rate at boss monsters, so this should be no exception.

File: ITEM.tsv [hunk 1]
```diff
@@ -6786,16 +6786,16 @@ ITEM_20150918_006785	Bomb with a high power output. It is extremely loud.
...
 ITEM_20150918_006792	Hauberk's soul divided itself in order to escape.
-ITEM_20150918_006793	A spectrum sphere created by Kyupol Daiba to keep watch on demons. It can be used to find the hiding Hauberk.
+ITEM_20150918_006793	A spectrum sphere created by Kupole Daiva to keep watch on demons. It can be used to find the hiding Hauberk.
 ITEM_20150918_006794	A rune containing the Goddess Vakarine's power. It absorbs the souls of demons.
 ITEM_20150918_006795	A trinket left by adventurers who entered the Demon Prison. It seems they were killed by demons.
-ITEM_20150918_006796	A token fragment dropped from the Lada Seal. Evidence that Dionys spread riots after he lost his ego.
-ITEM_20150918_006797	A claw found stuck in a demon. It looks like it was attacked by Dionys.
-ITEM_20150918_006798	Kyupol Sigita says that other demons will take this tooth as a warning.
+ITEM_20150918_006796	A token fragment dropped from the Lada Seal. Evidence that Dionysusus spread riots after he lost his ego.
+ITEM_20150918_006797	A claw found stuck in a demon. It looks like it was attacked by Dionysusus.
```
Comment (terrapin47) [inline ITEM.tsv]: Dionysus

**Comment by Sunaries** [comment]
Good to merge.

@imcgames I just want to add that I have resolved the bug of Hauberk. 

**My solution**: I took out my companion from the character login screen and joined the world. Then I started to accept the quest again and this time I slowly attacked instead of attacking it fast. After his HP was around 25%, a magic circle appeared and a speech bubble and he was gone and the cutscene finally ended. 

I have no doubt that users want to deal high damage at a very fast rate at boss monsters, so this should be no exception.

**Comment by terrapin47** [comment]
:+1: Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

@imcgames I just want to add that I have resolved the bug of Hauberk. 

**My solution**: I took out my companion from the character login screen and joined the world. Then I started to accept the quest again and this time I slowly attacked instead of attacking it fast. After his HP was around 25%, a magic circle appeared and a speech bubble and he was gone and the cutscene finally ended. 

I have no doubt that users want to deal high damage at a very fast rate at boss monsters, so this should be no exception.

**Comment by Sunaries** [comment]
Good to merge.

@imcgames I just want to add that I have resolved the bug of Hauberk.

My solution: I took out my companion from the character login screen and joined the world. Then I started to accept the quest again and this time I slowly attacked instead of attacking it fast. After his HP was around 25%, a magic circle appeared and a speech bubble and he was gone and the cutscene finally ended.

I have no doubt that users want to deal high damage at a very fast rate at boss monsters, so this should be no exception.

**Comment by imcgames** [comment]
Hello Spadow,

I fully agreed with you, It looks much simplified and better. I will try to include the design refer to your idea.

And, thank you for the bug report. It should be fixed before the 2nd iCBT. I forwarded these bugs to our Dev team for the reference.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**